requirements.txt

```diff
@@ -0,0 +1,2 @@
+ansible==8.6.1
+black==23.11.0
```

Comment:
black is not a requirement of the testbed itself. It's only required by developers. I think it should be removed from the requirements.txt file.

Reply:
Thats correct, that was for simplification.
(Due to the much larger fingerprint of the other components, i added this for convenience)
I can remove it.